website/src/docs/compressor.md

```diff
@@ -43,6 +43,8 @@ uppy.use(Compressor, {
 })
 ```
 
+> **Note:** that _all_ these options are passed to Compressor.js.
```

Comment:
I think this can be more clear. You are now saying all the _documented_ options are passed to Compressor.js, but the truth is that anything you put in `this.opts` will be passed. Furthermore, you should link to the options and mention to what it is passed (the constructor?)

Reply:
Updated